base/src/main/java/org/aya/core/visitor/Expander.java

```diff
@@ -115,6 +116,19 @@ record Normalizer(@NotNull TyckState state) implements Expander {
         case ElimTerm.App app -> applyThoroughly(CallTerm::make, app);
         case ElimTerm.Proj proj -> ElimTerm.proj(proj);
         case IntroTerm.PartEl el -> partial(el);
+        case ElimTerm.PathApp app -> {
```

Comment:
This should be in `Expander`

Reply:
Yes!!!

Reply:
![image](https://user-images.githubusercontent.com/9902643/187016407-d113af3c-4e4d-43c5-a0bc-085cf3bcd221.png)